### ISSUE TITLE
"clear" command does not clear the terminal screen on windows 10

### ISSUE THREAD

**Issue by victorgau** [jupyterlab/jupyterlab#4060]
I am trying JupyterLab on windows 10 (home).  When I run clear on terminal, it did not clear the screen.
Is it a bug? Check the link for animated gif: https://imgur.com/cVWNO4J

Thanks!

**Comment by jasongrout** [comment]
Can you check if clear works in the terminal in the classic notebook? You can get to the classic notebook via the JupyterLab Help menu.

**Comment by victorgau** [comment]
No... it does not work... animated gif: https://imgur.com/juPD42F

**Comment by blink1073** [comment]
The command was recently renamed to `Reconnect`, because it is only reconnecting the websocket connection.  Clearing has to be done in the terminal itself as a command.

**Comment by jasongrout** [comment]
In the gifs above, clear is being typed in the terminal as a terminal command.

**Comment by blink1073** [comment]
Ah, my mistake.  This is a limitation of `pywinpty`, cf https://github.com/jupyter/terminado/issues/45.

**Comment by dileep021** [comment]
any update on this?

**Comment by blink1073** [comment]
Hi @dileep021, no update yet, unfortunately.

**Comment by nathaneastwood** [comment]
I am assuming that this isn't an OS issue, but if it is, this also happens on Windows 7 using Git Bash.

**Comment by vidartf** [comment]
What happens with the command `cls`?

**Comment by comsavvy** [comment]
Action has to be taken about this issue.
Am currently facing the same problem right now

**Comment by leej11** [comment]
I believe I am experiencing the same issue!

**Comment by Adriankhl** [comment]
As a workaround, you can do `reset`

**Comment by Discobotdevlolhehe** [comment]
cls and clear is a python command. If u have python then update it if not, install it
